.gitignore

```diff
@@ -46,3 +46,5 @@
 # test-dars
 test-dars
 
+# share keys folder
+keys
```

Comment:
EOF newline, not particularly important here, but elsewhere will trigger checkstyle issues.